libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,252 @@
+    ]),
+    value: ethers.BigNumber.from(0).toHexString(),
+    gas: ethers.BigNumber.from(1_000_000).toHexString(),
+    type: "0x0",
+    gasPrice: (await provider.getGasPrice()).toHexString(),
+    nonce: "0x" + (await provider.getTransactionCount(addressToImpersonateCloneX)).toString(16),
+    chainId: "0x" + (await provider.getNetwork()).chainId.toString(16),
+  };
+
+  const cloneXTxHash = await provider.send("eth_sendTransaction", [sendCloneX]);
+  await provider.send("anvil_stopImpersonatingAccount", [addressToImpersonateCloneX]);
+  await provider.waitForTransaction(cloneXTxHash);
+};
+
+const defaultNanoApp = { firmware: "2.2.3" as const, version: "1.10.4" as const };
+
+export const scenarioEthereum: Scenario<EvmTransaction> = {
+  name: "Ledger Live Basic ETH Transactions",
+  setup: async () => {
+    const [{ transport, onSignerConfirmation }] = await Promise.all([
+      spawnSpeculos(
+        "speculos",
+        `/${defaultNanoApp.firmware}/Ethereum/app_${defaultNanoApp.version}.elf`,
+      ),
+      spawnAnvil("https://rpc.ankr.com/eth"),
+    ]);
+
+    const provider = new providers.StaticJsonRpcProvider("http://127.0.0.1:8545");
+    const signerContext = (deviceId: string, fn: any): any => fn(new Eth(transport));
+
+    setCoinConfig(() => ({
+      info: {
+        status: {
+          type: "active",
+        },
+        gasTracker: {
+          type: "ledger",
+          explorerId: "eth",
+        },
+        node: {
+          type: "external",
+          uri: "http://127.0.0.1:8545",
+        },
+        explorer: {
+          type: "ledger",
+          explorerId: "eth",
+        },
+      },
+    }));
+
+    const currencyBridge = buildCurrencyBridge(signerContext);
+    const accountBridge = buildAccountBridge(signerContext);
+    const getAddress = resolver(signerContext);
+    const { address } = await getAddress("", {
+      path: "44'/60'/0'/0/0",
+      currency: getCryptoCurrencyById("ethereum"),
+      derivationMode: "",
+    });
+
+    const scenarioAccount = makeAccount(address, ethereum);
+
+    await setBlock();
+
+    await initUSDCAccount(provider, address);
+    await initBoredApeAccount(provider, address);
+    await initCloneXAccount(provider, address);
+
+    await getLogs();
+
+    return { currencyBridge, accountBridge, account: scenarioAccount, onSignerConfirmation };
+  },
+  beforeAll: account => {
+    expect(account.nfts?.length).toBe(2);
+  },
+  transactions: [
+    scenarioSendTransaction,
+    scenarioUSDCTransaction,
+    scenarioERC721Transaction,
+    scenarioERC1155Transaction,
+  ],
+  afterAll: account => {
+    expect(account.operations.filter(operation => operation.type === "OUT").length).toBe(2);
+    expect(
+      account.operations
+        .filter(operation => operation.type === "FEES")
+        .filter(
+          feesOperations =>
+            feesOperations.nftOperations?.find(nftOperation => nftOperation.type === "NFT_OUT"),
+        ).length,
+    ).toBe(2);
+  },
```

Comment:
And same logic here, since we checked every transaction for its impact on account, we can just check something like:
Account should now have X ops, none of them have failed and balance is now: Y (TBD if we need to check more as previous expects already did the job I think)